### DISCUSSION TITLE
How to load schema if tables do not exist in Database.

### DISCUSSION THREAD

**Discussion by DenuxPlays** [SeaQL/sea-orm#2018] [Q&A]
Hey,
Is there a simple way to create all non-existing tables for all entities?
I tried this:
```rs
schema.create_table_from_entity(Model).if_not_exists();
```
But got the following error:
![image](https://github.com/SeaQL/sea-orm/assets/83671398/758a73ca-4580-407a-9bbe-3900009fd596)

The model was generated.

**Comment by tyt2y3** [comment]
No, there isn't. The short explanation is abstract classes don't exist in Rust, you can't pass them around or store them in an array.

**Reply by DenuxPlays** [reply]
Okay...
But then what is the point of this example: https://www.sea-ql.org/SeaORM/docs/schema-statement/create-table/
```rs
 builder.build(&schema.create_table_from_entity(CakeFillingPrice)),
 ```

**Reply by tyt2y3** [reply]
You can do it one by one, but you can't *create all*.

**Reply by DenuxPlays** [reply]
Okay...
Should be okay as long as I do not have to define a schema **and** the entity.
Don't want to do double the work.
I guess I just can't pass the struct `Model` into the function as it needs to have all field populated right?
If so, how to I do it?
